client/my-sites/stats/features/modules/stats-comments/stats-comments.tsx

```diff
@@ -0,0 +1,109 @@
+import { StatsCard } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { comment } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import QuerySiteStats from 'calypso/components/data/query-site-stats';
+import { useShouldGateStats } from 'calypso/my-sites/stats/hooks/use-should-gate-stats';
+import { useSelector } from 'calypso/state';
+import {
+	isRequestingSiteStatsForQuery,
+	getSiteStatsNormalizedData,
+} from 'calypso/state/stats/lists/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import EmptyModuleCard from '../../../components/empty-module-card/empty-module-card';
+import { INSIGHTS_SUPPORT_URL } from '../../../const';
+import Comments from '../../../stats-comments';
+import StatsCardSkeleton from '../shared/stats-card-skeleton';
+import type { StatsDefaultModuleProps, StatsStateProps } from '../types';
+
+interface CommentData {
+	authors: Array< {
+		label: string;
+		value: string;
+		iconClassName: string;
+		icon: string;
+		link: string;
+		className: string;
+		actions: Array< {
+			type: string;
+			data: boolean;
+		} >;
+	} >;
+	posts: Array< {
+		label: string;
+		value: string;
+		page: string;
+		actions: Array< {
+			type: string;
+			data: string;
+		} >;
+	} >;
+}
+
+const StatsComments: React.FC< StatsDefaultModuleProps > = ( { className } ) => {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId ) as number;
+	const statType = 'statsComments';
+	const moduleTitle = translate( 'Comments' );
+
+	// Use StatsModule to display paywall upsell.
+	const shouldGateStatsModule = useShouldGateStats( statType );
+
+	const isRequestingData = useSelector( ( state: StatsStateProps ) =>
+		isRequestingSiteStatsForQuery( state, siteId, statType, {} )
+	);
+	const data = useSelector( ( state: StatsStateProps ) =>
+		getSiteStatsNormalizedData( state, siteId, statType, {} )
+	) as CommentData;
+
+	const hasPosts = data?.posts?.length > 0;
+
+	return (
+		<>
+			{ ! shouldGateStatsModule && siteId && statType && (
+				<QuerySiteStats statType={ statType } siteId={ siteId } />
+			) }
+			{ isRequestingData && (
+				<StatsCardSkeleton
+					isLoading={ isRequestingData }
+					className={ className }
+					title={ moduleTitle }
+					type={ 3 }
+					withHero
```

Comment:
This property adds a large box above the title and is used to approximate things like a map for the Countries modules. Let's remove it from here.